src/vm/jitinterface.cpp

```diff
@@ -2596,39 +2595,27 @@ void CEEInfo::MethodCompileComplete(CORINFO_METHOD_HANDLE methHnd)
     EE_TO_JIT_TRANSITION();
 }
 
-// Given a module scope (scopeHnd), a method handle (context) and an metadata token,
-// attempt to load the handle (type, field or method) associated with the token.
-// If this is not possible at compile-time (because the method code is shared and the token contains type parameters)
-// then indicate how the handle should be looked up at run-time.
-//
-// See corinfo.h for more details
-//
-void CEEInfo::embedGenericHandle(
-            CORINFO_RESOLVED_TOKEN * pResolvedToken,
-            BOOL                     fEmbedParent,
-            CORINFO_GENERICHANDLE_RESULT *pResult)
+bool CEEInfo::computeNeedsRuntimeLookupAndHandleType(CORINFO_RESOLVED_TOKEN* pResolvedToken,
+                                                BOOL fEmbedParent,
+                                                CORINFO_GENERICHANDLE_RESULT* pResult  /* = NULL */,
+                                                CORINFO_METHOD_HANDLE* ppTemplateMethodHandle /* = NULL */)
 {
-    CONTRACTL {
+    CONTRACTL
+    {
         SO_TOLERANT;
         THROWS;
         GC_TRIGGERS;
         MODE_PREEMPTIVE;
     } CONTRACTL_END;
 
```

Comment:
Reverted the addition of the computeNeedsRuntimeLookupAndHandleType

---
In reply to: [62793078](https://github.com/dotnet/coreclr/pull/4519#discussion_r62793078) [](ancestors = 62793078)